Mathlib/Tactic/Finiteness/Attr.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Floris van Doorn
 -/
 import Aesop
+import Mathlib.Init
```

Comment:
This means that all of `aesop` will be imported in `Init`: maybe there should be a wider discussion about this.

(In fact, I'm not sure that `aesop` has not already been imported with `finiteness`, actually.)

Reply:
I can confirm; I think you have it packwards: this file imports Mathlib.Init (now), and it imports aesop (and did before this PR).